safe-global/web-core
# feat: redesign Load Safe

## What it solves

Resolves #964 

## How this PR fixes it

- Existing components from `src/components/load-safe` were copied into the new structure within `src/components/new-safe/load` with minor layout adjustments
- The first and second step of the old flow are combined
- Removes the old routes `/open` and `/load` and their pages
- Uses the new routes `/new-safe/create` and `/new-safe/load` everywhere
- Restructures the `new-safe` directory

## ToDos

- [x] Move `OwnerRow` into the parent directory
- [x] Remove old load safe components inside `src/components/load-safe`
- [x] Find a solution for cut-off safe addresses in the new design

## How to test it

1. Open the Safe
2. Navigate to the welcome page
3. Click on "Add existing Safe"
4. Observe the new design
5. Navigate to a specific safe
6. Open the safe list sidebar and click on "Add Safe"
7. Observe being navigated to the load safe flow and the safe address being pre-filled

## Screenshots
<img width="807" alt="Screenshot 2022-12-20 at 14 59 24" src="https://user-images.githubusercontent.com/5880855/208684262-6ac3142f-cf10-4259-87a1-fcf659898b4e.png">
<img width="822" alt="Screenshot 2022-12-21 at 11 49 22" src="https://user-images.githubusercontent.com/5880855/208887838-3b47893d-1dfd-49db-9efc-ef5d4dcd23ff.png">
<img width="822" alt="Screenshot 2022-12-20 at 15 05 38" src="https://user-images.githubusercontent.com/5880855/208685522-44d88612-8562-4fa2-9be9-d1a9454016ee.png">
